index.html

```diff
@@ -755,13 +755,12 @@ <h3>Validate Algorithm</h3>
 
       <p class="note" title="Issuer validation is use case dependent">
 It is expected that a <a>verifier</a> will ensure that it trusts the issuer
-of a <a href="#bitstringstatuslistcredential">BitstringStatusListCredential</a>,
-as well as the issuer of the associated <a>verifiable credential</a>, before
-utilizing the information contained in either credential for further decision
-making purposes. Implementers are advised that the issuer of each credential
-might be different in use cases where the original issuer of the
-credential does not maintain whether or not that credential continues to be
-valid.
+of a <a>verifiable credential</a>, as well as the issuer of the associated
+<a href="#bitstringstatuslistcredential">BitstringStatusListCredential</a>,
+before utilizing the information contained in either credential for further
+decision making purposes. Implementers are advised that the issuers of these
+credential might be different in use cases where the original issuer of the
+<a>verifiable credential</a> does not maintain a record of its validity.
       </p>
```

Comment:
```suggestion
before using the information contained in either credential for further
decision making purposes. Implementers are advised that the issuers of these
credential might be different, such as when the original issuer of the
<a>verifiable credential</a> does not maintain a record of its validity.
      </p>
```

Reply:
Fixed in 612b47717dc6f70d90abdc424314bbf87b779e9d.